torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4400,6 +4422,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCPU=all_types_and_complex_and(torch.bool),
            dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_diag),
+    OpInfo('diag_embed',
+           dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16),
+           supports_out=False,
+           supports_forward_ad=True,
+           sample_inputs_func=sample_inputs_diagonal),
+    OpInfo('diagonal',
+           dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16),
+           supports_out=False,
```

Comment:
Surprise! I thought `method_tests` always had entries for operators which supported autograd.

I wonder what they do for operators which don't support autograd.

cc: @mruberry

Reply:
Where do you see no autograd support?

Reply:
Ouch! I saw it as `support_autograd`. My bad :sweat_smile: